Nightwatch/Program.fs

```diff
@@ -70,20 +75,20 @@ let private configureResourceRegistry() =
     Log.Information("Available resources: {0}", String.Join(", ", names))
     Registry.create resourceFactories
 
-let private readConfiguration path factories : Result<Resource seq, InvalidConfiguration seq> =
-    async {
+let private readConfiguration path factories : Task<Result<Resource seq, InvalidConfiguration seq>> =
+    task {
         let! resources = Configuration.read factories FileSystem.system path
         let (results, errors) = splitResults resources
         return
             if Seq.isEmpty errors
             then Ok results
             else Error errors
-    } |> Async.RunSynchronously
+    }
 
 let private run = function
 | Ok resources ->
     logFullVersion()
-    runScheduler resources
+    runScheduler resources |> ignore
```

Comment:
I think that we need a `synchronize` here.

Reply:
Or maybe return `Task` from this function and add a `synchronize` later in `main`.